ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/dot1x.cfg

```
@@ -16,6 +16,16 @@ dot1x system-auth-control
 dot1x protocol lldp bypass
 dot1x dynamic-authorization
 dot1x
+   !
+   supplicant profile Profile1
+      identity <user_id>
+      eap-method tls
+      passphrase 0 <password>
+      ssl profile PF1
+   !
+   supplicant profile Profile2
+      identity <user_id>
+      passphrase 7 <password>
```

This shows invalid on EOS CLI, we might have to pass encoded value here.
Do check this once.
```suggestion
      passphrase 7 1048080A01
```